### PR TITLE
zh-CN: update the translation of html element

### PR DESCRIPTION
**Pull request by zjffun**

### CODE REVIEW

File: files/zh-cn/web/html/element/index.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+## 表格内容
+
+这里的元素用于创建和处理表格数据。
+
+{{HTMLRefTable("HTML tabular data")}}
+
+## 表单
+
+HTML 提供了许多可一起使用的元素，这些元素能用来创建一个用户可以填写并提交到网站或应用程序的表单。详情请参阅 [HTML 表单指南](/zh-CN/docs/Web/Guide/HTML/Forms)。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/index.md:113]:
```suggestion
HTML 提供了许多可一起使用的元素，这些元素能用来创建一个用户可以填写并提交到网站或应用程序的表单。详情请参阅 [HTML 表单指南](/zh-CN/docs/Learn/Forms)。
```

File: files/zh-cn/web/html/element/index.md [hunk 2]
```diff
@@ -63,6 +63,33 @@ HTML 支持各种多媒体资源，例如图像、音频和视频。
 
 {{HTMLRefTable({"include":["HTML embedded content"], "exclude":["multimedia"]})}}
 
+## SVG 和 MathML
+
+你可以使用 {{SVGElement("svg")}} 或 {{MathMLElement("math")}} 将 [SVG](/en-US/docs/Web/SVG) 或 [MathML](/en-US/docs/Web/MathML) 内容直接嵌入 HTML 文档。
```
Comment (yin1999) [inline files/zh-cn/web/html/element/index.md:68]:
```suggestion
你可以使用 {{SVGElement("svg")}} 或 {{MathMLElement("math")}} 将 [SVG](/zh-CN/docs/Web/SVG) 或 [MathML](/zh-CN/docs/Web/MathML) 内容直接嵌入 HTML 文档。
```

**Review by yin1999 — APPROVED** [review]

**Comment by yin1999** [comment]
please squash the last three commit, then pin me for a review

**Comment by zjffun** [comment]
Thank you. I squashed commits, and [fix a link](https://github.com/mdn/translated-content/compare/17e8b681261ec9c82d55c7903278e93eb1ad69dc..b350e2fe392b453b315e8cb3de34d25c06d70455).

**Comment by yin1999** [comment]
Thanks a lot

**Pull request merged**